lms/security.py

```diff
@@ -170,10 +170,10 @@ def identity(self, request):
     def permits(self, request, _context, permission):
         try:
             # Getting lti_use here again for the potential exception
-            # side effect and allow us to return DeniedWithValidationError accordingly
+            # side effect and allow us to return DeniedWithException accordingly
             self._get_lti_user(request)
-        except ValidationError as err:
```

Comment:
This is a bit of a wart but extends a mechanism that we are already using and that I find useful, trying to treat exceptions that happen during authentication in a similar way the do in other parts of the app (via the exceptions views).